### ISSUE TITLE
More information about model

### ISSUE THREAD

**Issue by harindercnvrg** [bes-dev/stable_diffusion.openvino#149]
@bes-dev Can you please more information about the models used in this repository?

For example, 

1. what are the base models used?
2. What is the precision format of the current model?
3. What version of Stable diffusion is being used?

etc.

It would be helpful. Thanks.

**Comment by code2k13** [comment]
Yes, even I was trying to get this information. @bes-dev your model generates ridiculously cool images effortlessly. The exact  same prompt, generates not very great looking images with stable-diffusion-v1.4 or 1.5 using the StableDiffusionPipeline. This has something different I feel, not sure what it is. Doesn't look like the vanilla SD 1.4 or SD 1.5.
